content/docs/2.3/scalers/solace.md

```diff
@@ -0,0 +1,104 @@
++++
+title = "Solace PubSub+"
+availability = "2.3+"
```

Comment:
We have already shipped v2.3, can you please update this and move to the correct folder please?
```suggestion
availability = "2.4+"
```

Reply:
Actually it is already in v2.4 so let's delete this file.

Reply:
Done